guardian/newsletters-nx
# Hide migration creation date in table

## What does this change?

We are, arbitrarily, using 946684800 as the migration timestamp in the [migration scripts](https://github.com/guardian/newsletters-migration-tool). This change checks for that date in the table date formatting and returns unknown instead.


## How to test

Check it out, manually update some newsletter to have the creation date of `946684800` (a number, not a string). Now review in the launched newsletters table. It should state `Unknown` rather than Sunday aa Jan, 1970 or whatever it was. 

![Screenshot 2023-06-12 at 14 13 19](https://github.com/guardian/newsletters-nx/assets/3277259/0796545a-42b8-47a0-ad54-67871bdf3d08)
